aten/src/ATen/native/vulkan/gen_glsl.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import sys
+import os
+import re
+
+H_NAME = "glsl.h"
+CPP_NAME = "glsl.cpp"
+
+def findAllGlsls(path):
+  cmd = "find " + path + " -name \"*.glsl\""
```

Comment:
This isn't safe in Buck.  We need to either pass the list of glsl files on the command line or put them all in a file and pass a path to that.